plugin/wakatime.vim

```diff
@@ -126,13 +126,13 @@ let s:VERSION = '9.0.1'
                 let path = path . '.exe'
             endif
 
-            " Check for wakatime-cli installed via Homebrew
```

Comment:
We still need this for Mac users.

Reply:
If the user config is correct, there should be no problem.
If we are worried about failure,  we should set these paths:

> Homebrew
> macOS Intel: /usr/local/bin
> macOS ARM: /opt/homebrew/bin
> Linux: /home/linuxbrew/.linuxbrew/bin
> or: ~/.linuxbrew/bin